packages/py-gt4py/package.py

```diff
@@ -68,7 +73,7 @@ class PyGt4py(PythonPackage):
 
     # pytest fails with:
     #__main__.py: error: unrecognized arguments: --cov-config=setup.cfg
-    patch('patches/functional/rm_cov_config_arg.patch', when='@functional')
+    patch('patches/functional/rm_cov_config_arg.patch', when='@main')
```

Comment:
Can you check if this patch is still needed?
Because right now we do not apply it for v1.0.1.

Reply:
Not needed anymore